e2e/libs/setting/setting.py

```diff
@@ -70,10 +90,26 @@ def set_backupstore(self):
                         self.SETTING_BACKUP_TARGET_CREDENTIAL_SECRET,
                         backupsettings[1],
                         retry=False)
+                    value = self.get_setting(
```

Comment:
I believe we no longer need to maintain `set_backupstore` and `reset_backupstore` in `setting.py`, since backup target has been moved from `setting` to `backup target CR`. these 2 functions have become unreachable.